styles/Button.module.css

```diff
@@ -11,13 +11,36 @@
   cursor: pointer;
 }
 
-.default_button:hover {
+.primary_button:hover {
   color: var(--text-default);
   background-color: transparent;
   border: 1px solid var(--border-default);
 }
 
-.default_button:active {
+.primary_button:active {
+  box-shadow: 2px 2px 5px #808080;
+}
+
+.secondary_button {
+  padding: 10px 16px;
+  width: 102px;
+  height: 35px;
+  background-color: transparent;
+  color: var(--text-default);
+  border-radius: 6px;
+  border: 1px solid var(--border-default);
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.secondary_button:hover {
+  color: var(--text-default);
+  background-color: transparent;
```

Comment:
Button hover has the same background colour as the normal background colour

Reply:
Fixed the hover effect & remove the invert one